### PR TITLE
[Docs]: Lifecycle hook example improvement

### PR DESCRIPTION
**Pull request by jainanuj0812**
## PR Type
Docs

```
Documentation content changes
```

## What is the current behavior?
**https://angular.io/guide/lifecycle-hooks
Live example for lifecycle hooks in documentation is not working as expected.

Below is the example link
https://stackblitz.com/angular/jlqeyrgaoaq

Clicking the 'Create PeekABooComponent' not logging the lifecycle hooks below it on UI.**

Issue Number: 22279

## What is the new behavior?
After clicking the 'Create PeekABooComponent' the logs should be viewed below it.

## Does this PR introduce a breaking change?
```
No
```

### CODE REVIEW

**Comment by jainanuj0812** [comment]
@IgorMinar , I have made the changes and tested it. Working as expected.

**Comment by splincode** [comment]
@jainanuj0812 You did not correctly name the commit, you just will not be accepted

See: https://github.com/angular/angular/blob/master/CONTRIBUTING.md

You need also usage `git rebase` for collapse in one commit

File: aio/content/examples/lifecycle-hooks/src/app/peek-a-boo-parent.component.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export class PeekABooParentComponent {
     this.hasChild = !this.hasChild;
     if (this.hasChild) {
       this.heroName = 'Windstorm';
-      this.logger.clear(); // clear log on create
+      this.hookLog.length = 0 // clear log on create
```
[the remark this answers is not in the document]
Comment (jainanuj0812) [reply aio/content/examples/lifecycle-hooks/src/app/peek-a-boo-parent.component.ts]: Yes, but as per @IgorMinar comment on my last commit I have submitted this PR, please review this one- https://github.com/angular/angular/pull/23201

[the remark this answers is not in the document]
Comment (petebacondarwin) [reply aio/content/examples/lifecycle-hooks/src/app/peek-a-boo-parent.component.ts]: I think @IgorMinar was suggesting making changes to the `logger.clear()` method no?

[the remark this answers is not in the document]
Comment (jainanuj0812) [reply aio/content/examples/lifecycle-hooks/src/app/peek-a-boo-parent.component.ts]: okay, I misunderstood, I will update this MR

Thanks @petebacondarwin

[the remark this answers is not in the document]
Comment (jainanuj0812) [reply aio/content/examples/lifecycle-hooks/src/app/peek-a-boo-parent.component.ts]: I am closing this PR and creating a new one

**Comment by jainanuj0812** [comment]
Closing and creating a new one!

**Pull request closed without merging**